Trailheads/src/classes/AnimalsCalloutTest.cls

```diff
@@ -0,0 +1,48 @@
+@isTest
+private class AnimalsCalloutTest {
+
+    @isTest static  void testGetCallout() {
+        // Create the mock response based on a static resource
+        StaticResourceCalloutMock mock = new StaticResourceCalloutMock();
+        mock.setStaticResource('GetAnimalResource');
+        mock.setStatusCode(200);
```

Comment:
Dlaczego na mocku masz te rzeczy do ustawienia? 
Pewnie trailhady - musisz mi zacząć znaczyć kod co jest z TH a co nie, albo gdzie mnie więcej coś zmieniałeś. Będzie prościej - bo część z moich uwag to są uwagi typu 'można to zrobić lepiej niż na th' a nie 'zjebaeś'